nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -293,6 +293,10 @@ SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "{{ secret_key }}")
 # CELERY_TASK_SOFT_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_SOFT_TIME_LIMIT", str(5 * 60)))
 # CELERY_TASK_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", str(10 * 60)))
 
+# Port for prometheus metric HTTP server running on the celery worker
+# Set this to 0 to disable it.
+# CELERY_WORKER_PROMETHEUS_PORT = int(os.getenv("NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORT"), 0)
```

Comment:
This seems wrong by comparison to `settings.py` - shouldn't it be the same logic as above, i.e. a comma-separated list?

Reply:
```suggestion
# CELERY_WORKER_PROMETHEUS_PORT = []
# if os.getenv("NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORT"):
#     CELERY_WORKER_PROMETHEUS_PORT = [
#         int(value) for value in os.getenv("NAUTOBOT_CELERY_WORKER_PROMETHEUS_PORT").split(",")
#     ]
```